dash-renderer/src/APIController.react.js

```diff
@@ -65,11 +82,24 @@ const UnconnectedContainer = props => {
         content = <div className="_dash-error">Error loading dependencies</div>;
     } else if (appLifecycle === getAppState('HYDRATED')) {
         renderedTree.current = true;
+
         content = (
-            <TreeContainer
-                _dashprivate_layout={layout}
-                _dashprivate_path={[]}
-            />
+            <DashContext.Provider value={provider.current}>
+                <TreeContainer
+                    _dashprivate_error={error}
+                    _dashprivate_layout={layout}
+                    _dashprivate_loadingState={getLoadingState(
+                        layout,
+                        [],
+                        loadingMap
+                    )}
```

Comment:
The loadingState is calculated by the parent component instead of through the store `connect` - also when not loading, the value is `false` instead of `{ is_loading: false }` - this is handled in the TreeContainer to remain consistent with the existing contract.